client/src/pages/Home.jsx

```diff
@@ -11,12 +13,12 @@ const Home = () => {
   
   const [map,setMap] = useState(/** @type google.maps.Map*/ (null));
   const center = {lat:0,lng:0};
-  const [markerPosition, setMarkerPosition] = useState({lat:44,lng:-80});
+  const [markerPosition, setMarkerPosition] = useState({lat:0,lng:0});
   const [markerKey,setMarkerKey] = useState(0);
   const [coordinates,setCoordinates] = useState(null);
   const [showCoordinates,setShowCoordinates] = useState(false);
 
-  const onClickMap = (e) => {
```

Comment:
Any reason you stopped using the arrow function here? It should be the preferred style in newer JS code